packages/admin/blocks-admin/src/blocks/factories/createListBlock.tsx

```diff
@@ -333,35 +359,39 @@ export function createListBlock<T extends BlockInterface, AdditionalItemFields e
                                                                             });
                                                                         }}
                                                                         visibilityButton={
-                                                                            canChangeVisibility ? (
-                                                                                <IconButton onClick={() => toggleVisible(data.key)} size="small">
-                                                                                    {data.visible ? (
-                                                                                        <Visible color="secondary" />
-                                                                                    ) : (
-                                                                                        <Invisible color="action" />
-                                                                                    )}
-                                                                                </IconButton>
-                                                                            ) : (
-                                                                                <Tooltip
-                                                                                    title={
+                                                                            <Tooltip
+                                                                                disableHoverListener={disableTooltipHoverListener}
```

Comment:
Why do we need this?

Reply:
so that the tooltip doesn't show up when it's not needed.

Reply:
I don't think that's necessary since the title isn't set anyway:

> Tooltip title. Zero-length titles string, undefined, null and false are never displayed.

https://mui.com/material-ui/api/tooltip/#tooltip-prop-title

Reply:
You're right. Removed that and the variable.

[3f1ddc1](https://github.com/vivid-planet/comet/pull/1865/commits/3f1ddc1f93a73d51174f4eac77b4786c3ecdbf45)